dev-tools/validation/test_check_proto_time_fields.py

```
@@ -0,0 +1,99 @@
+import io
+import unittest
+from contextlib import redirect_stderr
+from pathlib import Path
+import importlib.util
+
+
+SCRIPT_PATH = Path(__file__).resolve().parent / "check-proto-time-fields.py"
+
+
+def _load_checker_module():
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_load_checker_module`

(ANN202)